api/types.go

```diff
@@ -232,6 +232,10 @@ type EthereumSpecific struct {
 	GasLimit          *big.Int                               `json:"gasLimit"`
 	GasUsed           *big.Int                               `json:"gasUsed"`
 	GasPrice          *Amount                                `json:"gasPrice"`
+	L1Fee             *big.Int                               `json:"l1Fee,omitempty"`
+	L1FeeScalar       string                                 `json:"l1FeeScalar,omitempty"`
+	L1GasPrice        *Amount                                `json:"l1GasPrice,omitempty"`
+	L1GasUsed         *big.Int                               `json:"l1GasUsed,omitempty"`
```

Comment:
L1 fee data returned on optimism transaction receipts (needed to calculate total transaction fee L2 + L1)